packages/tocco-ui/src/Pagination/Pagination.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react'
+import _debounce from 'lodash/debounce'
+import './styles.scss'
+
+/**
+ *  Pagination component
+ */
+class Pagination extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      currentPage: 1,
+      totalPages: Math.ceil(props.totalRecords / props.recordsPerPage)
+    }
+
+    this.callOnPageChanged = _debounce(this.callOnPageChanged, 800)
+  }
+
+  onLastPage = () => this.state.currentPage >= this.state.totalPages
+  onFirstPage = () => this.state.currentPage <= 1
```

Comment:
this.state.currentPage === 0  or this.state.currentPage === 1? In the function `handleInputUpdate` the value will be set to 0 if the input is less than 0 so I guess the index of the first page has to be zero?